samples/subsys/usb/dfu/prj.conf

```diff
@@ -3,8 +3,12 @@ CONFIG_GPIO=y
 CONFIG_ARC_INIT=n
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DFU_CLASS=y
 CONFIG_FLASH=y
-CONFIG_SOC_FLASH_QMSI=y
+CONFIG_IMG_MANAGER=y
+CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_SYS_LOG=y
 CONFIG_SYS_LOG_USB_DRIVER_LEVEL=1
 CONFIG_SYS_LOG_USB_DEVICE_LEVEL=1
+CONFIG_TEXT_SECTION_OFFSET=0x200
```

Comment:
Remove this line it is not needed with `CONFIG_BOOTLOADER_MCUBOOT`

Reply:
@jfischer-phytec-iot Can you remove this line so we can get this merged? Looks like there is no issue with DFU itself, but rather with the Nordic driver.

Reply:
I will do, but I would like to try the page by page erase with NRF52 SoC before this PR will be merged.